TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/geometry/Pose2d.java

```diff
@@ -0,0 +1,98 @@
+package org.firstinspires.ftc.teamcode.util.geometry;
```

Comment:
I'm a little confused about this series of classes; aren't these all already available via some of the FIRST libraries? Same with the command stuff?